src/http/actions/actions.dependents.ts

```diff
@@ -11,10 +11,12 @@ import {
   _customerDependentRealtionshipTableName
 } from '../../models/types/model.customerDependentRelationship';
 import {db} from '../core/core.db';
-import { SQLQueryResponse,SQLRunner } from '../core/build/core.build.runner.sql';
+import { SQLQueryResponse, SQLRunner } from '../core/build/core.build.runner.sql';
+import { SQLInsertResponse } from '../../models/types/gen/gen.SQLResponse';
 
-// * SQL Runner to perform MYSQL Requests
-const _runner = new SQLRunner(db, _tableName);
+// * SQL Runners to perform MYSQL Requests
+const _dependentsRunner = new SQLRunner(db, _dependentTableName);
+const _relationshipRunner = new SQLRunner(db, _customerDependentRealtionshipTableName);
```

Comment:
We work mailly with dependents table here and with relationship table too.
Using SQLRunners for both tables reduces the SQL query chains length.

Reply:
@Alcazar87 please check if it's ok for you.